addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.yelo" name="Yelo" version="1.0.0" provider-name="shycoderX">
```

Comment:
I'm not sure about this project, but you don't need to change the version. This will be done when a new release is made.

Reply:
Just put it back to 1.0.0

Reply:
Correct, releases are usually decoupled from features/fixes.